GOVERNANCE.md

```diff
@@ -0,0 +1,26 @@
+# restic project governance
+
+## Overview
+
+The restic project uses a governance model commonly described as Benevolent
+Dictator For Life (BDFL). This document outlines our understanding of what this
+means. It is derived from the [i3 window manager project
+governance](https://raw.githubusercontent.com/i3/i3/next/.github/GOVERNANCE.md).
+
+## Roles
+
+* user: anyone who interacts with the restic project
+* core contributor: a handful of people who have contributed significantly to
+  the project by any means (issue triage, support, documentation, code, etc.).
+  Core contributors are recognizable via GitHub’s "Member" badge.
+* BDFL: a single individual who makes decisions when consensus cannot be
```

Comment:
maybe expand `BDFL` abbreviation? I had to google it to make sure I knew what it meant precisely.

Reply:
oh, good point, thanks!

Reply:
Hm. It was already explained in the section above, but now it's in there twice. Doesn't hurt, I'd say.